tests/python/pytorch/graphbolt/impl/test_ondisk_dataset.py

```diff
@@ -61,6 +61,53 @@ def test_OnDiskDataset_TVTSet_exceptions():
             _ = gb.OnDiskDataset(test_dir).load()
 
 
+def test_OnDiskDataset_TVTSet_ItemSet_names():
```

Comment:
Instead creating a new test case, the name attribute should be added for all other test cases.

Reply:
I don't it's necessarily required as the name attributes parse/use logic are fully covered in the newly added testcase. No need to test it other existing testcases which will be redundant.

Reply:
Test cases are also used by reader as an example of usage, it is better to align with the recommend way.

Reply:
added.